app/src/main/java/org/oppia/app/home/HomeActivityController.kt

```diff
@@ -12,7 +13,7 @@ class HomeActivityController @Inject constructor(private val activity: AppCompat
     activity.setContentView(R.layout.home_activity)
     activity.supportFragmentManager.beginTransaction().add(
       R.id.home_fragment_placeholder,
-      HomeFragment()
+      StateFragment()
```

Comment:
Revert this to `HomeFragment()` itself. so that this PR is ready for merge. By now in all PRs it is understood that we will have to change this to `StateFragment` to test your code. Also, better you write code which is merge-ready and in that if there are some steps which needs to be performed to check this PR mention that in the description of PR.

Reply:
okay